src/integ_test/java/games/strategy/engine/lobby/server/db/MutedMacControllerIntegrationTest.java

```diff
@@ -59,8 +59,12 @@ public void testMuteMacUpdate() {
   }
 
   private Instant muteMacForSeconds(final long length) {
-    final Instant muteEnd = length == Long.MAX_VALUE ? null : Instant.now().plusSeconds(length);
+    final Instant muteEnd = length == Long.MAX_VALUE ? null : nanosIgnored(Instant.now().plusSeconds(length));
```

Comment:
Just a suggestion for the "real" fix...  Having tests depend on wall time (i.e. `Instant#now()`) is generally a Bad Thing.  We should try to refactor these tests to use fixed `Instant`s of our choosing.  That probably means we need to be able to inject an `Instant` into the controller because I'm guessing it's using `Instant#now()`, as well.